### PR TITLE
Added JS question about composite data types

### PR DESCRIPTION
**Pull request by not-a-ethan**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

#14 

We have a question about primitive data types so why no composite types as well?

### CODE REVIEW

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1866,6 +1866,17 @@ const javascriptQuiz = [
     Explanation:
       "JavaScript has 7 primitive data types. Boolean, null, undefined, Number, BigInt, String and Symbol",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_Types#data_structures_and_types"
+  },
+  {
+    Question:
+      "In Javascript, whitch of the following is not a composite data type?",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1872]:
```suggestion
      "In Javascript, which of the following is NOT a composite data type?",
```

File: src/data/javascript-quiz.ts [hunk 2]
```diff
@@ -1866,6 +1866,17 @@ const javascriptQuiz = [
...
+    Answer: "String",
+    Distractor1: "Array",
+    Distractor2: "Class",
+    Distractor3: "Object",
+    Explanation:
+      "In JS a composite data type is defined as not being an object as well as having no properties or methods.",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1878]:
```suggestion
      "In JavaScript, a composite data type is defined as not being an object as well as having no properties or methods.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by not-a-ethan** [comment]
@jdwilkin4 All edits look good. I accepted the changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**